### ISSUE TITLE
Is there a better strategy than `doubleArrowButtonClickedBooleanProperty`?

### ISSUE THREAD

**Issue by zepumph** [phetsims/build-a-nucleus#113]
Promoting TODO to issue:

https://github.com/phetsims/build-a-nucleus/blob/b2d3cad640977c748299d2dde3c9b1dc7e171273/js/common/view/BANScreenView.ts#L786-L792

I couldn't think of any, but I'd like to brainstorm with @Luisav1. Please let me know if I can help out here.

**Comment by zepumph** [comment]
Tagging https://github.com/phetsims/build-a-nucleus/issues/93

**Comment by zepumph** [comment]
My main thought was to have some listener keep track of the associated particles, either in the doubleArrowButton listener, or from the "does this atom exist" Property, but those seem weird to me. 

Something else I feel like we should address is the `this.previousNeutronCount < neutronCount` part of the logic too. It seems a shame that we have 2 hacky ways of handling the "does not form" case: one for the double add, and one for the single add.

**Comment by zepumph** [comment]
@Luisav1 and I brainstormed how we may be able to support this better:

1. What if we kept track of the "current additions" where ever we added particles, then we already had that list if we needed to remove them. This is not ideal because it would touch at least 3 spots where additions occur (query parameters, dragging, increment buttons.
2. Having the particleAtom keep track of this data by keeping track of the most recent particle added. This doesn't handle the "double" add case.
3. Trying to have `previousProtonCount` and `previousNeutronCount` handle the double case, but this doesn't work because of how animations mean they are added to the particle at different times (so both counts will never be different from the actual count at the same time).

We are ready to close this issue now, and are glad to have tried out a few options.
